tests/main/cifs-home/task.yaml

```diff
@@ -0,0 +1,131 @@
+summary: Test that snaps still work when /home is a CIFS mount
+
+details: |
+    Snapd now contains a feature where CIFS-mounted /home (or any sub-directory)
+    initializes a workaround mode where all snaps gain minimal amount of network
+    permissions sufficient for CIFS to operate.
+
+backends: [-autopkgtest]
+
+# Limit the test to systems that have samba fairly new samba.
+systems:
+  - ubuntu-22.04-*
+  - ubuntu-24.04-*
+
+prepare: |
+    # We don't expect to leak the test-remote user, group or mount.
+    tests.cleanup defer NOMATCH 'test-remote' /etc/passwd
+    tests.cleanup defer NOMATCH 'test-remote' /etc/group
+    tests.cleanup defer NOMATCH 'test-remote' /proc/self/mountinfo
+
+    # Install a package with additional kernel modules, so that we can mount cifs.
+    case "$SPREAD_BACKEND" in
+      google)
+        case "$SPREAD_SYSTEM" in
```

Comment:
you can do instead
if os.query is-ubuntu-ge 22.04; then ...

Reply:
I realised the whole check is pointless at this time, so I reduced it to just simply doing what I wanted in the inner part.